v20.04/Dockerfile.amd64

```diff
@@ -9,8 +9,24 @@ LABEL maintainer="ownCloud GmbH <devops@owncloud.com>" \
   org.opencontainers.image.source="https://github.com/owncloud-docker/server" \
   org.opencontainers.image.documentation="https://github.com/owncloud-docker/server"
 
-ADD owncloud.tar.bz2 /var/www/
+############
+# imagemagick-7 upgrade to add HEIC support to the thumbnailer. HEIC was added in 6.9.11 as seen in
+# ubuntu 22.04, whereas ubuntu 20.04 has 6.9.10
+# See https://doc.owncloud.com/server/next/admin_manual/installation/manual_installation/manual_imagick7.html
+##
```

Comment:
Please dont add such long comments, this is git where you can use the commit search and git blame.